mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/repositories/TiltakstypeRepository.kt

```diff
@@ -201,34 +225,36 @@ class TiltakstypeRepository(private val db: Database) {
     fun getAllMedDeltakerregistreringsinnhold(): List<TiltakstypeEksternDto> {
         val tiltakstyper = getAll(paginationParams = PaginationParams(nullableLimit = 1000))
 
-        return tiltakstyper.second.map { tiltakstypeAdminDto ->
-            val deltakerRegistreringInnhold = getDeltakerregistreringInnholdByTiltakskode(tiltakstypeAdminDto.arenaKode)
-            TiltakstypeEksternDto(
-                id = tiltakstypeAdminDto.id,
-                navn = tiltakstypeAdminDto.navn,
-                arenaKode = tiltakstypeAdminDto.arenaKode,
-                registrertIArenaDato = tiltakstypeAdminDto.registrertIArenaDato,
-                sistEndretIArenaDato = tiltakstypeAdminDto.sistEndretIArenaDato,
-                fraDato = tiltakstypeAdminDto.fraDato,
-                tilDato = tiltakstypeAdminDto.tilDato,
-                rettPaaTiltakspenger = tiltakstypeAdminDto.rettPaaTiltakspenger,
-                status = tiltakstypeAdminDto.status,
-                deltakerRegistreringInnhold = deltakerRegistreringInnhold,
-            )
-        }
+        return tiltakstyper.second
+            .map { tiltakstypeAdminDto ->
+                val deltakerRegistreringInnhold = getDeltakerregistreringInnholdByArenaKode(tiltakstypeAdminDto.arenaKode)
+                TiltakstypeEksternDto(
+                    id = tiltakstypeAdminDto.id,
+                    navn = tiltakstypeAdminDto.navn,
+                    tiltakskode = tiltakstypeAdminDto.tiltakskode,
+                    arenaKode = tiltakstypeAdminDto.arenaKode,
+                    registrertIArenaDato = tiltakstypeAdminDto.registrertIArenaDato,
+                    sistEndretIArenaDato = tiltakstypeAdminDto.sistEndretIArenaDato,
+                    fraDato = tiltakstypeAdminDto.fraDato,
+                    tilDato = tiltakstypeAdminDto.tilDato,
+                    rettPaaTiltakspenger = tiltakstypeAdminDto.rettPaaTiltakspenger,
+                    status = tiltakstypeAdminDto.status,
+                    deltakerRegistreringInnhold = deltakerRegistreringInnhold,
+                )
+            }
     }
 
-    private fun getDeltakerregistreringInnholdByTiltakskode(tiltakskode: String): DeltakerRegistreringInnholdDto? {
+    private fun getDeltakerregistreringInnholdByArenaKode(arenaKode: String): DeltakerRegistreringInnholdDto? {
```

Comment:
Kunne vi migrert dette innholdet til å være koblet mot nye `tiltakskode` i stedet for `arena_kode`? Det er kun gruppetiltakene som har dette behovet.

Kan godt gjøres som en egen oppgave/PR om det er mer praktisk.

Reply:
Det fungerer ikke fordi tiltakskode ikke er unique... Så den kan ikke refereres med foreign key (kunne fortsatt måtte kalt metoden med den nye, men usikker på om det er noe bedre)

Reply:
Denne er gjort nå